test/IdTableHelpersTest.cpp

```diff
@@ -129,16 +143,47 @@ TEST(IdTableHelpersTest, createRandomlyFilledIdTableWithoutGeneratos) {
         }));
   };
 
-  IdTable result{createRandomlyFilledIdTable(5, 5, 0, 0, 10)};
+  // Sample request for the overload, that takes a single `JoinColumnAndBounds`.
+  IdTable result{
+      createRandomlyFilledIdTable(5, 5, JoinColumnAndBounds{0, 0, 10})};
   generalIdTableCheck(result, 5, 5, true);
   checkColumn(result, 0, 0, 10);
 
-  result = createRandomlyFilledIdTable(50, 58, 0, 30, 42);
+  result = createRandomlyFilledIdTable(50, 58, JoinColumnAndBounds{0, 30, 42});
   generalIdTableCheck(result, 50, 58, true);
   checkColumn(result, 0, 30, 42);
+
+  // Empty std::vector for the overload with the , who takes a vector of
+  // `JoinColumnAndBounds`.
```

Comment:
More explicitly would be:
`No join columns with explicit generators are specified, so all columns are filled randomly`.

Reply:
Done, see [e290909](https://github.com/ad-freiburg/qlever/pull/1016/commits/e290909efd3c584c1e8b42f67f6740729961503d).